.github/workflows/post_build.yml

```diff
@@ -2,39 +2,40 @@ name: Post Build
 
 concurrency: ${{ github.ref }}
 
-on:
+on: 
   workflow_dispatch:
-  
-    
+
 jobs:
   build:
     runs-on: ubuntu-latest
 
     env:
-      RESOURCE_GROUP_NAME: aqovia-devbox
-      IDENTITY_NAME: aqovia-devcenter-identity
-      GALLARY_NAME: aqovia_compute_gallery
-      DEVCENTER_NETWORK_CONNECTION_NAME: aqovia-devcenter-network-connection
-      PROJECT_NAME: aqovia-devbox-project
-      POOL_NAME: aqovia-devbox-project-pool
-      DEFFINITION_NAME: aqovia-devbox-definition
+      RESOURCE_GROUP_NAME: dlr-devbox
+      IDENTITY_NAME: dlr-devcenter-identity
+      GALLARY_NAME: dlr_compute_gallery
+      DEVCENTER_NETWORK_CONNECTION_NAME: dlr-devcenter-network-connection
+      PROJECT_NAME: dlr-devbox-project
+      POOL_NAME: dlr-devbox-project-pool
+      DEFFINITION_NAME: dlr-devbox-definition
       DEFFINITION_SKU: general_i_8c32gb256ssd_v2 # sku of image (for getting sku list run this command "az devcenter admin sku list --query '[].name' --output table")
       DEFFINITION_IMAGE: Win11-DevBox # name of image (the same name as folder in ./images)
-      DEVCENTER_NAME: aqovia-devcenter
-      DEVCENETER_VNET_NAME: aqovia-devcenter-vnet
+      DEVCENTER_NAME: dlr-devcenter
+      DEVCENETER_VNET_NAME: dlr-devcenter-vnet
       DEVCENETER_SUBNET_NAME: default
+      AZURE_TENANT_ID : ${{ vars.AZURE_TENANT_ID }}
+      AZURE_CLIENT_ID : ${{ vars.AZURE_CLIENT_ID }}
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Login to Azure
-        run: az login --service-principal -u ${{ secrets.AZURE_CLIENT_ID }} -p ${{ secrets.AZURE_CLIENT_SECRET }} --tenant ${{ secrets.AZURE_TENANT_ID }}
-          
+        run: az login --service-principal -u "${{ vars.AZURE_CLIENT_ID }}" -p "${{ secrets.AZURE_CLIENT_SECRET }}" --tenant "${{ vars.AZURE_TENANT_ID }}"
```

Comment:
Can you remove `=` signs from similar command in `.github/workflows/build_images.yml`?

Reply:
done

Reply:
I still see:
![image](https://github.com/DLR-PI/dev-box-images/assets/46093/8f9955ed-cf12-404d-a04d-dba9ae964582)

Reply:
Think theres a sync issue, 

heres the commit for the equal sign
https://github.com/DLR-PI/dev-box-images/pull/1/commits/561464b01416044d56a831e3d9d380ec887f479c